### DISCUSSION TITLE
have some issue when loading custom lsp  server via Mason

### DISCUSSION THREAD

**Discussion by diandian2710** [NvChad/NvChad#1499] [Q&A]
Hey I am new to nvchad, after read the doc and fork others custom config to test. i spend few hours trying to install LSP server via PackerCompile+MasonInstallAll but it doesn't work. 

<img width="1924" alt="Screen Shot 2022-08-21 at 10 13 32 pm" src="https://user-images.githubusercontent.com/32948486/185790337-166e90f2-d117-4a3e-9f3d-256e89be7d7d.png">
<img width="1272" alt="Screen Shot 2022-08-21 at 10 13 49 pm" src="https://user-images.githubusercontent.com/32948486/185790347-b119a1a3-6cd2-458e-8773-48bf52a68f9c.png">
<img width="1695" alt="Screen Shot 2022-08-21 at 10 14 18 pm" src="https://user-images.githubusercontent.com/32948486/185790358-38b5cee4-82cb-4a64-8473-a2ffc04313f6.png">

<img width="1715" alt="Screen Shot 2022-08-21 at 10 17 55 pm" src="https://user-images.githubusercontent.com/32948486/185790496-dc15fc5a-a8a2-4dc3-b1b0-f43a894c98ed.png">
(html-lsp was installed manually, all of the preinstalled lsp servers are not reflected)

**Comment by siduck** [comment]
do packercompile
